src/deprecated.js

```diff
@@ -29,6 +29,9 @@ jQuery.fn.extend( {
 		} else {
 			jQuery.ready( true );
 		}
+},
+	nodeName: function( elem, name ) {
+		return elem.nodeName && elem.nodeName.toLowerCase() === name.toLowerCase();
```

Comment:
Only one thing left in my mind. Import `src/core/nodeName.js` in this file and set `nodeName: nodeName` rather than writing the function twice.